description.md

```diff
@@ -1,82 +1,104 @@
-AppMap is a runtime code analysis platform and runtime data-enhanced AI coding assistant.
+# AppMap for JetBrains IDEs
 
-AppMap will automatically generate and visualize code-linked runtime data and traces about your Java, Python, Node.js, or Ruby application without needing to add custom instrumentation or OTEL spans.
+AppMap is a developer tools platform powered by runtime analysis and AI. It's composed of three components:
 
-Code traces and visualizations from AppMap show how your code really works, including APIs, database queries, web service request flows, security architecture, and more.
+* Navie - AI code architect.
+* AppMap data - Runtime traces of your application runtime.
+* AppMap diagrams - Interactive visualizations of AppMap data.
 
-AppMap’s AI coding assistant, Navie, will use data and visualizations to co-develop new features, fix complex bugs, optimize code performance and keep your code secure.
+By using AppMap data, Navie is the first AI code architect with the context to understand how your app works when it runs, like a principal engineer or software architect. AppMap data provides Navie with accurate information about application behavior, APIs, database queries, and more, providing valuable context so you can ask more challenging questions and get better answers.
 
-AppMap and Navie will help you:
+![image](https://github.com/getappmap/appmap-intellij-plugin/assets/1229326/ed36497e-d21d-4b79-be18-5fe4756f4649)
```

Comment:
@jansorg Can we include images in the plugin description?

Reply:
yes, that's possible. But I recommend a static URL and not something GitHub could remove at any time. The marketplace could show the image even after the GitHub page changed.

Reply:
Image links updated.

Reply:
Image links updated @jansorg @dustinbyrne